src/clr-angular/utils/focus-trap/focus-trap.directive.spec.ts

```diff
@@ -148,7 +199,7 @@ describe('FocusTrap', () => {
 
 @Component({
   template: `
-        <a href="#">Not in form</a>
+        <ng-container *ngIf="mainFocusTrap">
```

Comment:
Nitpick: You don't need the `ng-container`, you can just put the `*ngIf` on the host of the focus trap directly.